students/BrandonHenson/Lesson7/test_html_render.py

```diff
@@ -0,0 +1,69 @@
+# Brandon Henson
+# 5/5/18
+# Lesson 07 HTML Renderer
+import unittest
+import html_render as hr
+import os
+from io import StringIO
+cwd = os.getcwd()
+
+
+class TestHtmlRender(unittest.TestCase):
+
+    def test_element(self):
+        test_element = hr.Element('some stuff')
+        self.assertEqual(test_element.content, ["some stuff"])
+
+    def test_append(self):
+        test = hr.Element('some stuff in here')
+        test.append('some more stuff in here')
+        self.assertEqual(test.content, ['some stuff in here', 'some more stuff in here'])
+
+    def test_render(self):
+        element = hr.Element("Test")
+        element.tag = 'html'
+        f = StringIO()
+        element.render(f)
+        self.assertEqual(f.getvalue(), "<html>\n" + element.indent + "Test\n</html>\n")
+        element.tag = 'body'
```

Comment:
Same thing here: this is overriding the tag attribute on the `Element` instance instead of instantiating a new `Body` instance which would already have the tag attribute set to 'body'